index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+   <link rel="stylesheet" href="style.css">
+  <title>Personal Portfolio</title>
+</head>
+<body>
+   <header>
+      <nav>
+         <div class="logo"><a href="#">I<span>.J</span></a></div>
```

Comment:
- [ ] As stated by the last reviewer, The logo isn't a part of the `nav` and i will suggest that you take it out and place it within the `header` rather than within the `nav`. This will come in handy in further milestones when you begin to use text contents within the application for desktop view 👍
- [ ] Also, seem you introduced a new bug. The image in your logo is not the appropriate one as of last milestine. Kindly fix this 👍 

## New bug:
![image](https://github.com/Isaac-ej/Portfolio_Website/assets/50990925/c2dbe31b-b78e-4017-b856-c23fd43db2d2)